EverythingCmdPal/Helpers/SettingsManager.cs

```diff
@@ -84,8 +81,8 @@ public class SettingsManager : JsonSettingsManager
             Resources.sendto_description,
             "notepad.exe,$P$");
 
-        public int SortOption => int.Parse(_sortOption.Value ?? "14", CultureInfo.InvariantCulture);
-        public uint Max => uint.Parse(_max.Value ?? "10", CultureInfo.InvariantCulture);
+        public int SortOption => int.TryParse(_sortOption.Value, NumberStyles.Integer, CultureInfo.InvariantCulture, out int result) ? result : 14;
```

Comment:
`_sortOption` is guaranteed to be an int, so `TryParse` isn't really needed. `Parse` is sufficient.